bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/BasePage.html

```diff
@@ -22,5 +22,6 @@
 
         <pre style="display:none" wicket:id="code-internal"></pre>
         <wicket:container wicket:id="footer-container"/>
+        <wicket:container wicket:id="scripts-container"/>
```

Comment:
What's the difference with `footer-container` ?
Why do we need the second ?

Reply:
`domReady` scripts were not rendered for this page :(

According to my debugging:
`domReady` scripts goes to `scripts` container

So I have added
```
        add(new HeaderResponseContainer("scripts-container", "scripts"));

................

        <wicket:container wicket:id="scripts-container"/>

```

Reply:
Let's simplify it by using `"footer-container"` at https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/11ad1e53a3fdf5e16195e2b794d7bfabf550ee91/bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/WicketApplication.java#L162 and remove the new containers (java and html).

Reply:
Done!
Thanks for the hint :)